titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 1_
```markdown
## Titanic Dataset
```
test

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._

Reply:
my reply